### PR TITLE
Add vacuum settings for v2

### LINKED ISSUE #4814: Investigate and add autovacuum settings back for v2 partitioning
### Description

Currently if we add the autovacuum settings for v2 partitioning , the importer throws an error.  As in the image.
Screen Shot 2022-11-01 at 10 11 28 PM

We need to investigate what is going on and add autovacuum settings back.

### Steps to reproduce

When you run `SPRING_PROFILES_ACTIVE=v2 ./mvnw spring-boot:run -pl hedera-mirror-importer` it throws the above error.

### Additional context

_No response_

### Hedera network

other

### Version

latest main

### Operating system

_No response_

### PR DESCRIPTION
**Pull request by jnels124**
**Description**:
* add udf to configure vacuum settings 

**Related issue(s)**:

Fixes #4814 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Comment by jnels124** [comment]
I will add a test to check that the vacuum settings are added to newly created partitions but wanted to verify this approach before doing so.

File: hedera-mirror-importer/src/main/resources/db/migration/v2/R__autovacuum_insert_only_tables.sql [hunk 1]
```diff
@@ -1 +1,70 @@
...
+    end loop;
+end;
+$$
+language plpgsql;
 
+-- Vacuum settings for partitioned tables
+
+-- auto vacuum per each account balance snapshot if there are at least 10000 accounts' balance inserted
+select apply_vacuum_settings_for_partitioned_table('account_balance'::regclass, 'autovacuum_vacuum_insert_scale_factor=0, autovacuum_vacuum_insert_threshold=10000');
+-- based on 12 contract actions per smart contract transaction and max 300 TPS
```
Comment (steven-sheehy) [inline hedera-mirror-importer/src/main/resources/db/migration/v2/R__autovacuum_insert_only_tables.sql:30]: nit: Some new lines between tables would help readability.

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Review by xin-hedera — APPROVED** [review]
LGTM.

potentially future item, we may need to manually vacuum or vacuum freeze the closed partitions of insert only table in the maintenance job.

**Pull request merged**